text/metrics/0113-exemplars.md

```diff
@@ -0,0 +1,104 @@
+
+[#347](https://github.com/open-telemetry/opentelemetry-specification/pull/347) describes a set of standard aggregations in the metrics SDK. Here we describe how exemplars could be implemented for each aggregation.
+
+### Exemplar behaviour for standard aggregations
+
+#### HistogramAggregator
+
+The HistogramAggregator MUST (when enabled) maintain a list of exemplars whose values are distributed across all buckets of the histogram (there should be one or more exemplars in every bucket that has a population of at least one sample-able measurement). Implementations SHOULD NOT retain an unbounded number of exemplars.
+
+#### Sketch
+
+A Sketch aggregation SHOULD maintain a list of exemplars whose values are spaced out across the distribution. There is no specific number of exemplars that should be retained (although the amount SHOULD NOT be unbounded), but the implementation SHOULD pick exemplars that represent as much of the distribution as possible. (Specific details not defined, see open questions.)
+
+#### Last-Value
+
+Most (if not all) Gauges operate asynchronously and do not ever interact with traces. Since the value of a Gauge is the last measurement (essentially the other parts of an exemplar), exemplars are not worth implementing for Gauge.
+
+#### Exact
+
+The Exact aggregation will function by maintaining a list of `RawValue`s, which contain all of the information exemplars would carry. Therefore the Exact aggregation will not need to maintain any exemplars.
+
+#### Counter
+
+Exemplars give value to counter aggregations in two ways: One, by tying metric and trace data together, and two, by providing necessary information to re-create the input distribution. When enabled, the aggregator will retain a bounded list of exemplars at each checkpoint, sampled from across the distribution of the data. Exemplars should be sampled in a statistically significant way.
+
+#### MinMaxSumCount
+
+Similar to Counter, MinMaxSumCount should retain a bounded list of exemplars that were sampled from across the input distribution in a statistically significant way.
+
+#### Custom Aggregations
+
+Custom aggregations MAY support exemplars by maintaining a list of exemplars that can be retrieved by exporters. Custom aggregations should select exemplars based on their usage by the connected exporter (for example, exemplars recorded for Stackdriver should only be retained if they were recorded within a sampled trace).
+
+Exemplars will always be retrieved from aggregations (by the exporter) as a list of RawValue objects. They will be communicated via a
+
+```
+optional repeated RawValue exemplars = 6
+```
+
+attribute on the `metric_descriptor` object.
```

Comment:
Changed it to `Metric`, one level up from the actual aggregation data points as desired but not part of the descriptor (I think if we have metadata separate from the exemplar list that tells us how it was collected, it could go alone in the `MetricDescriptor`, but we don't right now so everything should be in `Metric`)

Reply:
Continue here: https://github.com/open-telemetry/opentelemetry-proto/pull/159